tutorials/cdn-fastly.md

```diff
@@ -174,14 +176,23 @@ calls back to your Cloud Storage bucket. And by using an Origin
 Shield that is connected to Google's network edge, you can increase
 the speed of the shield's origin pulls.
 
+It is normally recommended to use a multi-regional Cloud Storage
+[bucket location](https://cloud.google.com/storage/docs/locations), but when
+using an Origin, it is instead recomended to use a regional or dual-regional
+location. This ensures the data is always close to the Orign, improving
+performance, and reducing latency on cache fills.
+
+For example, the "[us-east4](https://cloud.google.com/storage/docs/locations)"
+region in Ashburn, Northern Virginia, is near to one of [Fastly's interconnects](https://docs.fastly.com/guides/integrations/google-cloud-storage#interconnect-locations), and can be selected as a
+**shielding** location.
+
 To configure an Origin Shield and connect it to Cloud Platform:
```

Comment:
Not part of the changes :) but to be consistent:

To configure an Origin Shield -> To configure an origin shield

FWIW, I'm recommending these changes based on the capitalization used in Fastly's documentation.

Reply:
Updated all instances from "Origin Shield" to "origin shield"